lib/src/widgets/reorder_flex/drag_state.dart

```diff
@@ -82,6 +82,8 @@ class DraggingState {
   /// The last computed size of the feedback widget being dragged.
   Size? feedbackSize = Size.zero;
 
+  GlobalObjectKey? draggingKey;
```

Comment:
Should we do
```
class DraggingState extends GlobalObjectKey {
```
To avoid collisions? Or just use `GlobalKey` since we don't use comparison on the `draggingKey`?